alarmbot/src/ui/get_recent_summary.py

```diff
@@ -0,0 +1,88 @@
+import requests
+from datetime import datetime
+from .utils import CONSTANTS, get_user_info
+
+
+def retrieve_stats(data, username, tag):
+    meta = data.get("meta")
+    stats = data.get("stats")
+
+    map = meta.get("map").get("name")
+    mode = meta.get("mode")
+    time = datetime.strptime(meta.get("started_at"), "%Y-%m-%dT%H:%M:%S.%fZ").strftime(
+        "%d/%m/%Y %H:%M:%S"
+    )
+
+    team = data.get("stats").get("team").lower()
+    enemy_team = "red" if team == "blue" else "blue"
+    rounds = data.get("teams")
+    team_rounds = rounds.get(team)
+    enemy_rounds = rounds.get(enemy_team)
+    total_rounds = team_rounds + enemy_rounds
+    result = (
+        "W"
+        if team_rounds > enemy_rounds
+        else "L"
+        if team_rounds < enemy_rounds
+        else "D"
+    )
+
+    agent = stats.get("character").get("name")
+    score = stats.get("score")
+    kills = stats.get("kills")
+    deaths = stats.get("deaths")
+    assists = stats.get("assists")
+    shots = stats.get("shots")
+    damage = stats.get("damage")
+
+    kda = f"{kills}/{deaths}/{assists}"
+    acs = str(round(score / total_rounds))
+    adr = str(round(damage.get("made") / total_rounds))
+    dmg_delta = str(round((damage.get("made") - damage.get("received")) / total_rounds))
+    kd = str(round(kills / deaths, 2))
+    hsr = "{:.0%}".format(
+        round(
```

Comment:
the round seems redundant as you're using `.format`.

Reply:
ok but format just truncates